src/main/java/com/uber/cadence/internal/sync/POJOActivityTaskHandler.java

```diff
@@ -115,6 +126,8 @@ public void addActivityImplementation(Object activity) {
               timeoutException.getTimeoutType(),
               dataConverter.toData(timeoutException.getDetails()));
     }
+
+    metricsScope.counter(MetricsType.ACTIVITY_EXEC_FAILED_COUNTER).inc(1);
```

Comment:
Should we increment this one as well in case of Error?

Reply:
I'm matching the go client behavior here. My understanding is that Error here is go's panic. Let me know if it's not true.